Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -1569,6 +1580,73 @@ - (IBAction)showImageChoices:(id)sender {
     [self beginTaskWithIdentifier:ImageChoicesTaskIdentifier];
 }
 
+# pragma mark - Image Capture
+- (id<ORKTask>)makeImageCaptureTask {
+    NSMutableArray *steps = [NSMutableArray new];
+    
+    {
+        ORKInstructionStep *step = [[ORKInstructionStep alloc] initWithIdentifier:@"begin"];
+        step.title = @"Hands";
+        step.image = [UIImage imageNamed:@"hands_solid"];
```

Comment:
Yes. I hadn't thought about `UIImageRenderingModeAlwaysOriginal`, but this way of dealing with it seems consistent.